routEar/__main__.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+from flask import Flask, request, Response
+from os import path
+
+if sys.platform == "win32":
+    from win10toast import ToastNotifier
+    notif = ToastNotifier()
+elif sys.platform == "linux":
+    from pydbus import SessionBus
+    bus = SessionBus()
+    notif = bus.get(
+        'org.freedesktop.Notifications',
+        '/org/freedesktop/Notifications'
+    )
+    notif_id = None
+else:
+    print("unsupported platform")
+    sys.exit(1)
+
+def notify( title='title', body='text', icon='python'):
+    if sys.platform == "win32":
+        toaster.show_toast(
+            title,
+            body,
+            threaded=True,
+            icon_path=path.join(next(i for i in routEar.__path__),'data', '%s.ico'%icon)

```

Comment:
@ahmubashshir This line is generating the following error: 
```
File "D:\Documents\PythonCodes\routerShit\routEar\__init__.py", line 60, in callback
    notify(message_line_1, message_line_2)
File "D:\Documents\PythonCodes\routerShit\routEar\__init__.py", line 29, in notify
    next(i for i in routEar.__path__), 'data', '%s.ico' % icon)
NameError: name 'routEar' is not defined
```

Reply:
I forgot that __path__ was a global constant :3

Reply:
fixed in #3